MC_Setup.cmd

```diff
@@ -85,6 +85,7 @@ put #var MC_IMBUE ROD
 put #var MC_IMBUE.MANA 50
 put #var MC_Enchanting_NOWO 0
 #put #var MC_FOCUS.WAND wood wand
+put #var MC_PREFERRED.ROOM.ARTIF NULL

```

Comment:
added a preferred room for Artif work.  This is to support my brazier variable next. If you have your own brazier, then you may want a different room than your work outside room... As you may need mana for imbue, but your outside room may be suited to foraging or something.